cli/flox/doc/flox-update.md

```diff
@@ -0,0 +1,56 @@
+---
+title: FLOX-UPDATE
+section: 1
+header: "flox User Manuals"
+...
+
+
+# NAME
+
+flox-update - update the global base catalog or an environment's base catalog
+
+# SYNOPSIS
+
+```
+flox [ <general-options> ] update
+     [--global | (-d=<path> | -r=<owner/name>)]
+```
+
+# DESCRIPTION
+
+Update the global base catalog if `--global` is specified,
+or update an environment's base catalog.
+
+The base catalog is a collection of packages used by various flox subcommands.
+
+The global base catalog provides packages for
+[`flox-search(1)`](./flox-search.md) and [`flox-show(1)`](./flox-show.md) when
+not using an environment,
+and it is used to initialize an environment's base catalog.
+
+An environment's base catalog provides packages for
+[`flox-search(1)`](./flox-search.md) and [`flox-show(1)`](./flox-show.md) when
+using that environment,
+and it provides packages for [`flox-install(1)`](./flox-install.md) and
+[`flox-upgrade(1)`](./flox-upgrade.md).
+
+Updating an environment's base catalog does not actually update the packages in
```

Comment:
Suggestion: "does not actually..." kind of sounds like it's a deficiency in the tool. I might reframe this as updating packages and updating base catalogs being two separate operations and you may want to update one or the other at different times. With flox you have the _flexibility_ to update one without updating the other, etc.

Reply:
Fixing